frontend/src/buttons/ConnectButton/ConnectButton.tsx

```diff
@@ -21,7 +22,17 @@ export const ConnectButton: React.FC<ConnectButtonProps> = ({
 }) => {
   const hidden = (connection && connection.active) || !service || service.state !== 'active'
   const connecting = !!(connection && connection.pid && !connection.active)
-  const connect = () => emit('service/connect', connection || newConnection(service))
+  const connect = () => {
+    let theConnection = connection || newConnection(service)
+    let context = {
+      connectionType: 'peer',
+      serviceId: service?.deviceID,
```

Comment:
There is a service id as well as a device id... the serviceId is not the DeviceId.

Reply:
change to .id